vernac/lemmas.ml

```diff
@@ -403,19 +314,9 @@ let start_lemma_com ~program_mode ~poly ~scope ~kind ?inference_hook ?hook thms
   start_lemma_with_initialization ?hook ~poly ~scope ~kind evd ~udecl recguard thms snl
 
 (************************************************************************)
-(* Admitting a lemma-like constant                                      *)
+(* Commom constant saving path                                          *)
 (************************************************************************)
 
-let check_anonymity id save_ident =
-  if not (String.equal (atompart_of_id id) (Id.to_string (default_thm_id))) then
-    user_err Pp.(str "This command can only be used for unnamed theorem.")
-
-(* Admitted *)
-let warn_let_as_axiom =
-  CWarnings.create ~name:"let-as-axiom" ~category:"vernacular"
-                   (fun id -> strbrk "Let definition" ++ spc () ++ Id.print id ++
-                                spc () ++ strbrk "declared as an axiom.")
```

Comment:
Not sure moving this stuff is a great idea, it will mostly confuse git blame.

Reply:
This is due to grouping things into their proper sections (and comments), I dunno.